tox.ini

```diff
@@ -58,7 +58,7 @@ allowlist_externals =
 deps =
     -r{toxinidir}/requirements/doc.txt
 commands =
-    doc8 --ignore-path docs/_build README.rst docs
+    # doc8 --ignore-path docs/_build README.rst docs
```

Comment:
accidental?
```suggestion
    doc8 --ignore-path docs/_build README.rst docs
```

Reply:
Oh, right. I couldn't build the docs with that in there because we aren't building docs properly in CI and there were syntax linting issues in some of the ADRs. I meant to fix that in a separate commit after everything else was passing. I'll do that now.

Reply:
Okay, this is fixed up. There's definitely more docs cleanup that needs to be done later as a followup.